docs/arcaflow/workflows/expressions.md

```diff
@@ -261,6 +261,7 @@ Functions:
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
 | `readFile(string)`                               | string |  Returns the contents of a file as a UTF-8 character string, given a file path string. Relative file paths are resolved from the Arcaflow process working directory. Shell environment variables are not expanded.                                                                                     |
+| `bindConstants(list[any], any)`                               | list[object] |  Returns a list of objects each containing two properties: an `item` property which contains the corresponding item from the list in the first parameter; and, a `constant` property which contains the value of the second input parameter. The ID of the return value schema is the concatenation of the schema name of the first parameter and the schema name of the second parameter.                                                                                 |
```

Comment:
I still really don't like this. Saying that the schema names are concatenated without describing *how* is of no value, and just complicates the awkwardly wrapped tabular description here. A link to the detailed description would be more compact and much more useful.

Reply:
I'm questioning whether the calling workflow has any need to know the schema ID of the value returned by `bindConstants()`.  If it doesn't (or, if it is really rare), then perhaps we can just drop that sentence from here an provide the info in a more detailed document elsewhere.

Reply:
removed and linked